include/fmt/base.h

```diff
@@ -145,6 +145,10 @@ import std;
 #elif defined(__cpp_nontype_template_args) && \
     __cpp_nontype_template_args >= 201911L
 #  define FMT_USE_NONTYPE_TEMPLATE_ARGS 1
+#elif defined(__clang_major__) && __clang_major__ >= 12 && \
+    __cplusplus >= 202002L
```

Comment:
Let's simplify this check to `FMT_CLANG_VERSION >= 1200 && __cplusplus >= 202002L`.

Reply:
Done